src/ext/java/org/opentripplanner/ext/transmodelapi/model/stop/StopPlaceType.java

```diff
@@ -328,7 +338,8 @@ public static GraphQLObjectType create(
             Integer departuresPerLineAndDestinationDisplay = environment.getArgument(
               "numberOfDeparturesPerLineAndDestinationDisplay"
             );
-            Duration timeRage = Duration.ofSeconds(environment.getArgument("timeRange"));
+            Integer timeRangeInput = environment.getArgument("timeRange");
```

Comment:
Is timeRangeInput guaranteed to be non-null?

Reply:
It has a non-null default value https://github.com/opentripplanner/OpenTripPlanner/pull/4383/files/6d739cda5c9ae117142f5a55ac2cbd20f02fca67#diff-bd545dc8f92c0e70da477aa981c09595f274d13b59d009b1aefc9a6892b53e76R273